cpp/modules/math.gl/core/src/core.h

```diff
@@ -123,12 +125,16 @@ auto operator!=(const Vector2<coord> &v1, const Vector2<coord> &v2) -> bool {
 ///////////////////////////////////////////////////////////
 //  Vector3
 
+// TODO(ilija@unfolded.ai): When using std140 memory layout in GLSL vec3 takes in 16bytes
+// We should either use vec3 in our shaders instead, or create a GPUVector3 class that we'd use instead
```

Comment:
You mean we should use vec4s? 

I'd prefer we learnt how to get the layout right.